main.go

```diff
@@ -62,11 +62,13 @@ func main() {
 
 	if !config.Cfg.DeployedInHub {
 		leaseReconciler := lease.LeaseReconciler{
-			KubeClient:           config.GetKubeClient(),
+			HubKubeClient:        config.GetKubeClient(config.Cfg.AggregatorConfig),
```

Comment:
@jlpadilla Is this the right config to use to connect to hub - to create lease if the managed cluster is OCP311?

Reply:
I'm confused by the 2 different clients and config.  Let's chat.

Reply:
Verified that the permissions of the hub kubeconfig are limited.
I wasn't allowed to create a lease in the default namespace.
```
E0708 19:17:11.545741       1 lease.go:80] Unable to get addon lease "default"/"search-collector" . error:leases.coordination.k8s.io "search-collector" is forbidden: User "system:open-cluster-management:cluster:jorge-managed:addon:search-collector:agent:zd9qb" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "default"
```

Reply:
Also validated that the lease can be created in the hub, but didn't test with OCP 3.11